agents/locobot/hello_teleop.py

```diff
@@ -72,46 +129,80 @@ def unstop_robot(sid):
 
 def test_command(sid, commands, data={"yaw": 0.1, "velocity": 0.1, "move": 0.3}, value=None):
     print(commands, data, value)
-    move = float(data['move'])
+    move_dist = float(data['move'])
     yaw = float(data['yaw'])
     velocity = float(data['velocity'])
     global mover
     if mover == None:
         return
     if value is not None:
-        move = value
+        move_dist = value
+
+    def sync():
+        time.sleep(10)
+        for i in range(50):
+            mover.get_rgb_depth()
 
     movement = [0.0, 0.0, 0.0]
     for command in commands:
         if command == "MOVE_FORWARD":
-            movement[0] += move
-            print("action: FORWARD")
+            movement[0] += float(move_dist)
+            print("action: FORWARD", movement)
+            mover.move_relative([movement], blocking=False)
         elif command == "MOVE_BACKWARD":
-            movement[0] -= move
-            print("action: BACKWARD")
+            movement[0] -= float(move_dist)
+            print("action: BACKWARD", movement)
+            mover.move_relative([movement], blocking=False)
         elif command == "MOVE_LEFT":
```

Comment:
is this better named 'turn left' and 'turn right'? As I understand, the semantic parser uses that semantic. @aszlam to confirm if move left is the same as turn left.

Reply:
"move left" would be interpreted as a left "strafe", not a left turn

Reply:
changing this also depends on changing locobot_agent.py.
I've made a note for myself to send in a separate patch to change all the instances of MOVE_LEFT -> TURN_LEFT

Just FYI, in the teleop, we hook up the textbox to a raw pattern-match, i.e. it doesn't invoke an interpreter or parser.